content/en/docs/tutorials/services/connect-applications-service.md

```diff
@@ -90,7 +90,7 @@ kubectl expose deployment/my-nginx
 service/my-nginx exposed
 ```
 
-This is equivalent to `kubectl apply -f` the following yaml:
+This is equivalent to `kubectl apply -f` in the following yaml:
```

Comment:
That command is not "in" the yaml file. The original statement is correct, but if you want to clarify, maybe something like the following would be better?

```txt
This is equivalent to running `kubectl apply -f` to a file containing the following yaml:
```

Reply:
We actually call them "manifests", not YAMLs